polkadot/runtime/parachains/src/scheduler.rs

```diff
@@ -203,6 +204,14 @@ impl<T: Config> Pallet<T> {
 		ValidatorGroups::<T>::decode_len().unwrap_or(0)
 	}
 
+	/// Claim queue len
+	fn expected_claim_queue_len(config: &SchedulerParams<BlockNumberFor<T>>) -> u32 {
```

Comment:
I chose `expected_*` because the real claim queue could have less entries (again for cores that don't have assignments or cores that don't have groups assigned). I'll update the doc comment